packages/walletconnect/src/provider.ts

```diff
@@ -134,19 +134,8 @@ export class WalletConnectProvider extends SignerProvider {
       this.updateNamespace(this.session.namespaces)
     } else {
       this.updateNamespace(this.session.namespaces)
-      await this.ping(this.session.topic)
     }
-  }
-
-  private async ping(topic: string) {
-    const timeout = new Promise<never>((_, reject) => {
-      setTimeout(() => reject(new Error('Auto connect timeout')), 5 * 1000)
-    })
-    await Promise.race([this.client.ping({ topic }), timeout]).catch((err) => {
-      this.session = undefined
-      this.account = undefined
-      return this.client.disconnect({ topic, reason: { code: 0, message: `Error: ${err}` } })
-    })
+    await this.client.core.relayer.messages.del(this.session.topic)
```

Comment:
Every time a dApp sends a request, walletconnect stores the request message in the `messages` store. It will only be deleted if it is disconnected or the session expires.